skglm/utils/data.py

```diff
@@ -123,13 +124,13 @@ def make_correlated_data(
         return X, Y, w_true
 
 
-def make_dummy_survival_data(n_samples, n_features, normalize=False, random_state=None):
+def make_dummy_survival_data(n_samples, n_features, normalize=False,
+                             X_density=1., random_state=None):
     """Generate a random dataset for survival analysis.
 
-    The design matrix ``X`` is generated according to standard normal,
-    the vector of time is chosen uniformly from ``[0, 10 * n_samples]``
-    without replacement, and the vector of censorship is drawn from a Bernoulli
-    with parameter 0.5.
+    The design matrix ``X`` is generated according to standard normal, the vector of
+    time ``tm`` is chosen uniformly from ``[0, 10 * n_samples]`` without replacement,
```

Comment:
Minor: for `tm`, you can sample from a Weibull distribution, this will yield more realistic samples.